src/quo2/components/dropdowns/old_button_style.cljs

```diff
@@ -0,0 +1,266 @@
+(ns quo2.components.dropdowns.old-button-style
+  (:require [quo2.foundations.colors :as colors]))
+;TODO: Dropdown needs to be implemented as its own component -
+;https://github.com/status-im/status-mobile/issues/16456
+
+(def blur-view
+  {:position :absolute
+   :top      0
+   :left     0
+   :right    0
+   :bottom   0})
+
+(defn before-icon-style
+  [{:keys [override-margins size icon-container-size icon-background-color icon-container-rounded?
+           icon-size]}]
+  (merge
```

Comment:
Just felt like sharing, you surely know this, but we can use `cond->` for this style of code. Might even perform better than `merge?`? Well, I haven't measured, probably a micro optimization anyway.

```clojure
(defn before-icon-style
  [{:keys [override-margins size icon-container-size icon-background-color icon-container-rounded?
           icon-size]}]
  (cond-> {:margin-left     (or (get override-margins :left)
                                (if (= size 40) 12 8))
           :margin-right    (or (get override-margins :right) 4)
           :align-items     :center
           :justify-content :center}
    icon-container-size
    (assoc :width icon-container-size :height icon-container-size)

    icon-background-color
    (assoc :background-color icon-background-color)

    icon-container-rounded?
    (assoc :border-radius (/ (or icon-container-size icon-size) 2))))
```

Reply:
Thanks @ilmotta! 🙌 will do